<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSiteTab</class>
 <widget class="QWidget" name="GeneralSiteTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="name_label">
     <property name="text">
      <string>PLACEHOLDER</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="description_label">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="description_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="subnets_label">
       <property name="text">
        <string>Subnets:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="subnets_list_widget"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
